### ISSUE TITLE
[BUG] Document rules and mapping

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/axe-windows#164]
Please check whether the bug has [already been filed](https://github.com/Microsoft/axe-windows/issues).

## Describe the bug
In this repo, there isnt a place where the rules and the WCAG/Section 508 mapping are documented

## Expected behavior

A table that says the rule name, description, standard and if its a warning or a failure

**Comment by ferBonnin** [comment]
this is resolved now. [Rules description](https://github.com/microsoft/axe-windows/blob/master/docs/RulesDescription.md)
